blockchain/v2/reactor.go

```diff
@@ -156,7 +157,7 @@ type blockApplier interface {
 // XXX: unify naming in this package around tmState
 // XXX: V1 stores a copy of state as initialState, which is never mutated. Is that nessesary?
```

Comment:
want to just fix this comment while youre here: 
```suggestion
// XXX: V2 stores a copy of state as initialState, which is never mutated. Is that nessesary?
```

Reply:
I think v1 is maybe appropriate here? I think it's referring to the fact that the reactor started with the v1 version, and it's unclear what the v1 reactor was using initialState for.